### PR TITLE
Use gmake for FreeBSD

### PR DESCRIPTION
**Pull request by matsduf**
* Updated Makefile.PL so that gmake is used for share/Makefile
* Updated installation instruction to include devel/gmake
* Updated MANIFEST to include yet another file under "inc"

### CODE REVIEW

**Comment by matsduf** [comment]
Please test this on Linux too.

File: MANIFEST [hunk 1]
```diff
@@ -9,6 +9,7 @@ docs/Translation-translators.md
 inc/Module/Install.pm
 inc/Module/Install/Base.pm
 inc/Module/Install/Can.pm
+inc/Module/Install/External.pm
```
Comment (pnax) [inline MANIFEST:12]: This file is not present on Ubuntu : 

```
$ make distcheck
"/usr/bin/perl" "-Iinc" "-MExtUtils::Manifest=fullcheck" -e fullcheck
No such file: inc/Module/Install/External.pm
```

**Review by pnax — COMMENTED** [review]
Where is the `inc/Module/Install/External.pm` from ? I doesn't seem to be present when building on Ubuntu.

File: MANIFEST (hunk 1 above)
Comment (matsduf) [reply MANIFEST:12]: I suspected that. It appeared when building on FreeBSD after adding
```
if ($^O eq "freebsd") {
    requires_external_bin 'gmake';
};
```
to Makefile.PL. I will investigate.

Comment (matsduf) [reply MANIFEST:12]: Now inc/Module/Install/External.pm is included by the second commit. Please review again.

**Review by mattias-p — APPROVED** [review]
I haven't tested it but it makes sense to me in principle.

**Comment by matsduf** [comment]
@pnax, can you test again?

**Review by pnax — APPROVED** [review]
This works on Ubuntu.

@mattias-p sorry I misclicked for the review (and I don't know how to reset it)

**Pull request merged**